smt/design_space/__init__.py

```diff
@@ -0,0 +1,55 @@
+import importlib
+
+spec = importlib.util.find_spec("smt_design_space")
+if spec:
+    HAS_SMTDesignSpace = True
```

Comment:
constant are all uppercase ie HAS_SMT_DESIGN_SPACE. I would prefer HAS_SMT_DESIGN_SPACE_EXT or HAS_DESIGN_SPACE_EXT for a shorter name.